docs/graph-algorithms/prim-algorithm.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: Prim Algorithm
+slug: /graph-algorithms/prim-algorithm
+---
+
+## Introduction
+
+The Prim's algorithm is a greedy algorithm. This algorithm is used to find a minimum spanning tree for a weighted undirected graph. This means it finds a subset of the edges that forms a tree that includes every vertex, where the total weight of all the edges in the tree is minimized.
+
+## Explanation
+
+From a given undirected graph find the path such that it covers all the edges and has minimum total weight.
```

Comment:
Instead of external images you can rely on text and some markdown diagrams to better convey your point.

Reply:
@HarshCasper markdown diagrams means??